docs/changelog.mdx

```diff
@@ -4,6 +4,42 @@ sidebar_label: 'Changelog'
 
 # Changelog
 
+## Release 0.0.15
+
+_November 21, 2023_
+
+:::info
+
+This version is a release preview, and it's **our biggest release yet!** Our product is in beta and we would love for developers to 
+try it out and provide feedback! Contact us at info@regression.gg for dedicated support and access.
+
+:::
+
+**New Features**
+
+- New welcome screen added to the SDK, with docs and **sample scene/bot** options
+- Introducing C# bots that run **directly in Unity** (reference materials [here](/studios/unity/unity-sdk/creating-bots/csharp/configuration))
+- Introducing [**Bot Marketplace**](/studios/platform_features/marketplace) (Beta), which provides ready-to-go bots and integration instructions for those bots
+- Introducing **Validation Dashboard** (Beta), which provides an interface to view validation runs for a bot, complete with screenshots, state information, and historical records of previous runs
+- Introducing [**Agent Builder**](/players/creating-bots/agent-builder) (Beta), which provides a fast, low-code way to develop bots
+- Improved debug utilities like RGGizmos to see what your bot is thinking
+- Bots can now be directly synced into your Unity projects
+- New tutorials and guides for getting started
+
+**General Improvements**
+
+- New homepage dashboard for game developers, showing the latest runs, bots, and guides
+- The SDK no longer depends on packages like Cinemachine
```

Comment:
We mention this, but we say this is the first time we have support for C# bots in unity above.  Is that contradictory/confusing ?  Maybe we omit this line

Reply:
Removed!